test/jit/test_save_load_for_op_version.py

```diff
@@ -425,3 +422,62 @@ def _helper(m, fn):
                 self.assertEqual(mr, hr)
 
         _helper(v3_mobile_module, current_mobile_module)
+
+    def test_versioned_linspace(self):
+        class Module(torch.nn.Module):
```

Comment:
Looks nice! Can we reuse the code from `fixtures_srcs`? Also is it possible to have similar `historic_linspace` function, as the test in `test_versioned_div_scalar`?

Reply:
we can't because that model won't work in the new runtime.